scripts/get_ct_accounts_from_lists.py

```diff
@@ -0,0 +1,74 @@
+import json
+import requests
+from collections import defaultdict
+
+lists_url = "https://api.crowdtangle.com/lists"
+
+headers = {
+    'x-api-token': "" , #Add CT dashboard token here
```

Comment:
Let's make this comment more explicit, maybe move it to the line above `headers = {` so people see it?

Reply:
And/or make the token say `"YOUR_TOKEN"`

Reply:
done.